Salary App/ViewController.swift

```diff
@@ -13,8 +13,11 @@ class ViewController: UIViewController, UITextFieldDelegate {
     
     @IBOutlet weak var salarySC: UISegmentedControl!
     
+    @IBOutlet weak var monthlySalaryLbl: UILabel!
```

Comment:
Rather than storyboards can we add all the UI programatically, use a stack view to hold all the views this will show you how to use stackviews for building UI components